packages/kbn-unified-field-list/src/components/field_stats/field_stats.tsx

```diff
@@ -7,13 +7,9 @@
  */
 
 import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
-import {
-  type DataView,
-  type DataViewField,
-  ES_FIELD_TYPES,
-  getEsQueryConfig,
-  KBN_FIELD_TYPES,
-} from '@kbn/data-plugin/common';
+import type { DataView, DataViewField } from '@kbn/data-plugin/common';
```

Comment:
could be ... 
```suggestion
import type { DataView, DataViewField } from '@kbn/data-views-plugin/common';
```
or we remove all those kind of DataView or Data plugin imports in a different PR

Reply:
Updated